_drafts/2018-04-26-deboguer-vos-applications-dockerisees-avec-phpstorm.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: post
+title: Déboguer vos applications dockerisées avec PhpStorm  
+excerpt: Aujourd'hui je vais prendre un peu de votre temps pour vous expliquer comment lancer vos tests unitaire et les déboguer avec PhpStorm, le tout sans avoir besoin d'installer php, phpunit ou xdebug sur votre machine ...
+authors:
+- rmasclef
+permalink: /fr/debug-run-phpunit-tests-using-docker-remote-interpreters-with-phpstorm/
+categories:
+    - CATEGORIE 1
```

Comment:
if someone has any idea of good categories

Reply:
tu peux mettre php, docker

Reply:
les catégories ne sont pas à jour